cli/cmd/plugin/ui/web/tanzu-ui/src/shared/services/awsDeployment.tsx

```diff
@@ -0,0 +1,118 @@
+    };
+    const getAwsRequestPayload = () => {
+        let awsClusterParams: AWSManagementClusterParams = {
+            awsAccountParams: {
+                profileName: awsState.data.PROFILE,
+                sessionToken: awsState.data.SESSION_TOKEN,
+                region: awsState.data.REGION,
+                accessKeyID: awsState.data.ACCESS_KEY_ID,
+                secretAccessKey: awsState.data.SECRET_ACCESS_KEY,
+            },
+            loadbalancerSchemeInternal: false,
+            sshKeyName: awsState.data.EC2_KEY_PAIR,
+            createCloudFormationStack: false,
+            clusterName: awsState.data.CLUSTER_NAME,
+            controlPlaneFlavor: awsState.data.CLUSTER_PLAN,
+            controlPlaneNodeType: awsState.data.CONTROL,
+            bastionHostEnabled: true,
+            machineHealthCheckEnabled: true,
+            vpc: {
+                cidr: awsState.data.VPC_CIDR,
+                vpcID: '',
+                azs: [
+                    {
+                        name: 'us-west-2a',
+                        workerNodeType: awsState.data.CLUSTER_WORKER_NODE_TYPE,
+                        publicSubnetID: '',
+                        privateSubnetID: '',
+                    },
+                ],
+            },
+            enableAuditLogging: false,
+            networking: {
+                networkName: '',
+                clusterDNSName: '',
+                clusterNodeCIDR: '',
+                clusterServiceCIDR: awsState.data.CLUSTER_SERVICE_CIDR,
+                clusterPodCIDR: awsState.data.CLUSTER_POD_CIDR,
+                cniType: 'antrea',
+            },
+            ceipOptIn: true,
+            labels: {},
+            os: {
+                name: 'ubuntu-20.04-amd64 (ami-0dd0327a3bfaa4dc8)',
+                osInfo: {
+                    arch: 'amd64',
+                    name: 'ubuntu',
+                    version: '20.04',
+                },
+            },
+            annotations: {
+                description: '',
+                location: '',
+            },
+            identityManagement: {
+                idm_type: IdentityManagementConfig.idm_type.NONE,
+            },
+        };
+        return awsClusterParams;
+    };
+    const deployOnAws = async () => {
+        const awsClusterParams: AWSManagementClusterParams =
+            getAwsRequestPayload();
+        try {
+            const configFileInfo: ConfigFileInfo =
+                await AwsService.applyTkgConfigForAws(awsClusterParams);
+            await AwsService.createAwsManagementCluster(awsClusterParams);
+            dispatch({
+                type: DEPLOYMENT_STATUS_CHANGED,
+                payload: {
+                    type: DeploymentTypes.MANAGEMENT_CLUSTER,
+                    status: DeploymentStates.RUNNING,
+                    provider: Providers.AWS,
+                    configPath: configFileInfo.path,
+                },
+            });
+        } catch (e) {
```

Comment:
please change the console output to something like:

`Error when calling config or create API: ${e}`